### ISSUE TITLE
Handling of saved configuration files

### ISSUE THREAD

**Issue by pgleeson** [openworm/sibernetic#80]
It would be nice to be able to save multiple snapshots of the current simulation. Currently, pressing s saves the config to ./configuration/snapshot/configuration_default. If it gets pressed again, the configuration_default file gets overwritten. Maybe append the file with the current date? Or use a file name based on the original config file loaded, e.g.

    demo1_09.02.34_19Aug2015
    worm_10.02.34_19Aug2015

Also, why is the configuration dir required when reloading, but not for loading one of the inbuilt config files:

    ./Release/Sibernetic -f demo1

but

    ./Release/Sibernetic -f ./configuration/snapshot/configuration_default

Maybe allow:

    ./Release/Sibernetic -f snapshot/configuration_default

**Comment by skhayrulin** [comment]
@pgleeson about dynamical name of snapshots I though about it but haven't implemented yet. I think it won't be difficult.
About path:  actually it's historically I stored all configuration in configuration folder so by default I'm always looking for the file with config in configuration folder. But main problem here is that I can't run custom configuration from other location not configuration folder so I decided next if file has no slashes (\ or /) in name Sibernetic try to find file in configuration folder and file has slash in name it means that user identifies absolute path to file with configuration.
